qcschema/dev/wavefunction/core_wavefunction.py

```diff
@@ -0,0 +1,34 @@
+"""
+(Effective) core (aka one-electron) Hamiltonian
+"""
+
+core_wavefunction = {}
+
+# core hamiltonian
+core_wavefunction["h_core_a"] = {
+    "type": "array",
+    "description": "Alpha-spin core (one-electron) Hamiltonian in the AO basis",
+    "items": {"type": "number"}
```

Comment:
Under `items` we might also want to specify `shape : {"nso", "nso"}` or similar? Since `shape` isn't a keyword it can pass through in the schema.

Reply:
When specifying `shape : {"nso", "nso"}` are we indicating that (e.g.) `h_core_a` is a 1D array of length nso^2 that represents a square matrix of size `nso`, or are we actually going to have `h_core_a` be a list of lists ([[1.0, 0.0],[0.0, 1.0]])?

Reply:
{"nao", "nmo"}